packages/comms/src/services/fileSpray.ts

```diff
@@ -13,6 +13,10 @@ export interface UpdateDFUWorkunitEx extends UpdateDFUWorkunitMinusWU {
 
 export class FileSprayService extends FileSprayServiceBase {
 
+    DFUWUFileEx(request: FileSpray.DFUWUFileRequest): Promise<string> {
```

Comment:
Should keep the `DFUWUFile` name (especially since the generated version is invalid).

Reply:
I haven't been able to find a solution to this that doesn't seem like working around the rules. I did come across this answer on SO, which seemed like a good breakdown https://stackoverflow.com/a/62515586. Specifically, the third part where they suggested making the extended type be generic with a default of the preferred type. I thought that might be a good enhancement to the wsdl generator?